lib/Twig/Environment.php

```diff
@@ -104,6 +104,9 @@ public function __construct(Twig_LoaderInterface $loader = null, $options = arra
         } else {
             @trigger_error('Not passing a Twig_LoaderInterface as the first constructor argument of Twig_Environment is deprecated since version 1.21.', E_USER_DEPRECATED);
         }
+        if (PHP_VERSION_ID >= 50600) {
+            require_once __DIR__.DIRECTORY_SEPARATOR.'twig_call_method.php';
```

Comment:
shouldn't this be done at the top of the Twig_Template file instead, as this is where the function is called ?